### PR TITLE
Add TimeDelta property type

### PR DESCRIPTION
**Pull request by phantomii**
**Added `TimeDelta` type to handle `datetime.timedelta` values with configurable min/max constraints.** Introduced a new `TimeDelta` class in `types.py` to represent and validate timedelta values, converting them to/from float-based seconds for serialization. The class includes support for OpenAPI specifications with `minimum` and `maximum` values, defaulting to system-specific limits calculated via `ctypes` (added as a dependency) to determine platform-agnostic boundaries.

**Enhanced `DateTime` type initialization** to accept optional `min_value` and `max_value` parameters, though implementation details are pending.

**Added comprehensive unit tests** in `test_types.py` to validate `TimeDelta` behavior, including edge cases (min/max boundaries), invalid inputs, and compatibility with the calculated `TIMEDELTA_INFINITY` constant. Tests ensure proper validation, serialization, and handling of system-specific limits.

**Fixed OpenAPI spec generation** for numeric types by replacing `INFINITI` with system float limits when generating `maximum`/`minimum` values, ensuring compatibility with OpenAPI standards.

### CODE REVIEW

File: restalchemy/dm/types.py [hunk 1]
```diff
@@ -41,6 +42,10 @@
     r"[0-9a-fA-F]{12}"
 )
 
+TIMEDELTA_INFINITY = size_bytes = (
```
Comment (gmelikov) [inline restalchemy/dm/types.py:45]: size_bytes isn't used anywhere.

Comment (phantomii) [inline restalchemy/dm/types.py:45]:
```suggestion
TIMEDELTA_INFINITY =  (
```
